web/blueprints/finance/__init__.py

```diff
@@ -869,7 +870,8 @@ def transactions_unconfirmed_json():
                         icon='fa-trash',
                         btn_class='btn-danger btn-sm'
                     )
-                ] if privilege_check(current_user, 'finance_change') else [],
+                ]
+            if privilege_check(current_user, 'finance_change') else [],
```

Comment:
This is incorrect/misleading indentation, because the `if` part only removes the `actions` entry, and no other columns of the row. 
Before I suggest something that's incorrect in the eyes of `black`/`darker`, just throw `darker` on this file.